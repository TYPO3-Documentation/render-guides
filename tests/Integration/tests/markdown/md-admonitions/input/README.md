# MarkDown admonitions

> [!NOTE]
> **Image and screenshot files** should be placed within `Images/MyGuidesFileName/` where `MyGuidesFileName` is your guide's filename without the final `.md`.
