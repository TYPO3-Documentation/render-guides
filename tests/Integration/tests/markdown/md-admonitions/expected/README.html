<!-- content start -->

<section class="section" id="markdown-admonitions">
            <h1>MarkDown admonitions<a class="headerlink" href="#markdown-admonitions" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">&nbsp;<i class="fa-solid fa-link"></i></a></h1>

<div class="admonition note" role="alert">
    <p class="admonition-title">Note</p>
         <strong>Image and screenshot files</strong> should be placed within <code>Images/MyGuidesFileName/</code> where <code>MyGuidesFileName</code> is your guide&#039;s filename without the final <code>.md</code>.
</div>
    </section>
        <!-- content end -->
